ldapclient/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+- name: Set fact with LDAP server IP address
+  ansible.builtin.set_fact:
+    ldap_server_ip: "{{ hostvars['ldap-server']['ansible_host'] }}"
```

Comment:
how is this going to be if we have 2 ldapservers?

Reply:
Is that a thing that will happen? How does this work in the ldap config? I can use the group to join some vars if I know how the config should look like

Reply:
I mean in the past both nisprint and backup ran NIS.   We want more than 1 LDAP server don't we?

Reply:
I don’t disagree. I just asked how the config would work

Reply:
I'm not sure. lol.